spec/models/message_spec.rb

```diff
@@ -9,13 +12,15 @@
     end
 
     it "requires a description presence" do
-      message = FactoryGirl.create :invalidmessage
+      attrs = FactoryGirl.attributes_for(:message, content: nil)
+      message = Message.create attrs
```

Comment:
@bennytseng come see me about this one. `FactoryGirl.attributes_for` shouldn't be used in this case, but you used it correctly above when making the post request.